spec/controllers/idv/in_person/verify_info_controller_spec.rb

```diff
@@ -116,6 +117,129 @@
         )
       end
     end
+
+    context 'tracks costs' do
+      let(:review_status) { 'pass' }
+      let(:async_state) { instance_double(ProofingSessionAsyncResult) }
+      let(:adjudicated_result) do
+        {
+          context: {
+            stages: {
+              threatmetrix: {
+                transaction_id: 1,
+              },
+              resolution: {
+                transaction_id: 'resolution-mock-transaction-id-123',
+                vendor_name: 'ResolutionMock',
+              },
+              residential_address: {
+                transaction_id: 'resolution-mock-transaction-id-123',
+                vendor_name: 'ResolutionMock',
+              },
+              state_id: {
+                transaction_id: 'state-id-mock-transaction-id-456',
+                vendor_name: 'StateIdMock',
+              },
+            },
+          },
+        }
+      end
+
+      before do
+        allow(controller).to receive(:load_async_state).and_return(async_state)
+        allow(async_state).to receive(:done?).and_return(true)
+      end
+
+      context 'when same address as id is true and in aamva jurisdiction' do
+        let(:pii) do
+          { same_address_as_id: 'true' }
+        end
+
+        it 'adds costs to database' do
+          allow(subject).to receive(:pii).and_return(pii)
```

Comment:
This is overall a great addition to the specs! We could think about adding a similar one to the remote verify_info_controller_spec, not necessarily in this PR. That also brings up thoughts about how the cost calculation should be part of progressive proofer rather than verify info concern, but again, not a problem for this PR.

Let's cut down on the amount of stubbing so we can have more confidence in our specs. You already have a let for pii_from_user at the top, and it's hooked into user_session on line 20. If the suggestion below doesn't work, you might have to specify the whole thing: `user_session['idv/in_person'][:pii_from_user]`...

Same suggestion for all the specs below to set `:same_address_as_id` to the desired value.
```suggestion
        it 'adds costs to database' do
          pii_from_user[:same_address_as_id] = true
```

Reply:
I definitely think that's a good call out about moving the file where the costs are added to the db. Matt W. also made that point. It would definitely make sense to move it as it would be a lot more clear what's happening as the cost is occurring in the progressive proofer. I'll add a ticket for that. 

I'll update the spec, this is definitely cleaner.

Reply:
I updated the spec and found I didn't need to update the value of `same_address_as_id` in the it block

Reply:
[Ticket](https://cm-jira.usa.gov/browse/LG-11943) to move the cost tracking to the progressive proofer